src/main/java/macrobase/analysis/result/OutlierClassificationResult.java

```diff
@@ -0,0 +1,4 @@
+package macrobase.analysis.result;
+
+public abstract class OutlierClassificationResult {
```

Comment:
Do we think we'll want to overload this a bunch?

Off the top of my head, the basic members we'll almost always want:
 - The datum
 - boolean: isOutlier
 - maybe the score...